agent/pom.xml

```diff
@@ -2376,7 +2376,7 @@
                 <artifactId>maven-verifier-plugin</artifactId>
                 <configuration>
                     <verificationFile>verification.xml</verificationFile>
-                    <failOnError>true</failOnError>
+                    <failOnError>${verificationFailOnError}</failOnError>
                 </configuration>
```

Comment:
Perhaps a better alternative would be to have an empty verification file (noverification.xml), default the filename to verification.xml and reset it int the skipITs profile.

Reply:
I did change it for empty verification file. The only thing is that the `tests/pom.xml` uses two files and I defined two separate maven properties. I can adjust if needed differently.

Reply:
The new version looks fine.